src/PrestaShopBundle/Translation/Provider/TranslationFinderTrait.php

```diff
@@ -53,12 +56,25 @@ public function getCatalogueFromPaths($paths, $locale, $pattern = null)
         if (null !== $pattern) {
             $finder->name($pattern);
         }
-        $translationFiles = $finder->files()->notName('index.php')->in($paths);
+
+        try {
+            $translationFiles = $finder->files()->notName('index.php')->in($paths);
+        } catch (\InvalidArgumentException $e) {
+            throw new FileNotFoundException(
```

Comment:
I wanted to do that too, but isn't a BC break?

Reply:
I don't think so, because that exception is a subclass of the one that was being thrown before.